packages/d3fc-annotation/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "devDependencies": {
     "babel-polyfill": "^6.16.0",
-    "canvas": "2.3.1",
```

Comment:
This was causing npm install to fail on node 12, since the older canvas version doesn't have prebuilt binaries for this node version. It attempts to build from source which needs a bunch of other native dependencies to succeed, so I bumped the version up.